app/scripts/modules/core/src/presentation/formsy/BasicFieldLayout.tsx

```diff
@@ -0,0 +1,50 @@
+import * as React from 'react';
+import { IFormFieldLayoutProps } from './formFieldLayout';
+
+/**
+ * A Form Field Layout component for Formsy Form Field components
+ *
+ * Accepts four react elements as props (_label, _input, _help, _error) and lays them out using bootstrap grid.
+ *
+ * +----------------div.form-group----------------------------+
+ * |                 +-------------div.col-md-9--------------+|
+ * |<label.col-md-3> |+-------------------------------------+||
+ * |                 ||input element                        |||
+ * |                 |+-------------------------------------+||
+ * |                 |                                       ||
+ * |                 |<help element>                         ||
+ * |                 |                                       ||
+ * |                 |<validation element>                   ||
+ * |                 |                                       ||
+ * |                 +---------------------------------------+|
+ * +----------------------------------------------------------+
+ */
+export class BasicFieldLayout extends React.Component<IFormFieldLayoutProps, {}> {
+  constructor(props: IFormFieldLayoutProps) {
+    super(props);
+  }
+
+  public render() {
+    const { _label, _input, _help, _error, showRequired, showError } = this.props;
```

Comment:
oh wait, I just took them out and now I remember why I put the underscores.
This interface extends the HTMLAttributes, which defines `label` as a `string` only

![screen shot 2017-05-19 at 3 10 47 pm](https://cloud.githubusercontent.com/assets/2053478/26268699/5e3df6ac-3ca5-11e7-98f5-c893cbe7317e.png)